llvm/cmake/modules/AddLLVM.cmake

```diff
@@ -635,6 +635,11 @@ function(llvm_add_library name)
     set(ARG_STATIC)
   endif()
 
+  if(EMSCRIPTEN)
```

Comment:
Perhaps add `set(ARG_STATIC)` too?

Perhaps add a comment here:  "Emscripten's support for dynamic linking is limited so force all libraries to be statically linked"

Reply:
Doing it right away !

Reply:
Done.